ui/elastic/src/App.js

```diff
@@ -0,0 +1,260 @@
+  }
+
+  render() {
+    return (
+      <div className="App">
+        <SearchkitProvider searchkit={searchkit}>
+          <Layout>
+
+            <TopBar>
+              <SearchBox
+              autofocus={true}
+              searchOnChange={true}
+              placeholder="Search a build ID, eg: 201706*"
+              queryFields={["build.id"]}/>
+            </TopBar>
+
+            <LayoutBody>
+              <SideBar>
+                <RefinementListFilter field="target.version"
+                            title="Version"
+                            id="versions"
+                            size={1000}
+                            operator="OR"
+                            orderKey="_term"
+                            orderDirection="desc"
+                            listComponent={ItemCheckboxList}
+                            translations={{"All":"All versions"}}/>
+                <RefinementListFilter field="target.platform"
+                            title="Platform"
+                            id="platform"
+                            size={1000}
+                            operator="OR"
+                            orderKey="_term"
+                            listComponent={ItemCheckboxList}
+                            translations={{"All":"All platforms"}}/>
+                <RefinementListFilter field="target.channel"
+                            title="Channel"
+                            id="channel"
+                            size={1000}
+                            operator="OR"
+                            orderKey="_term"
+                            listComponent={ItemCheckboxList}
+                            translations={{"All":"All channels"}}/>
+                <RefinementListFilter field="target.locale"
+                            title="Locale"
+                            id="locale"
+                            size={1000}
+                            operator="OR"
+                            orderKey="_term"
+                            listComponent={ItemCheckboxList}
+                            translations={{"All":"All locales"}}/>
+              </SideBar>
+
+              <LayoutResults>
+
+                <ActionBar>
+                  <ActionBarRow>
+                    <HitsStats/>
+                    <SortingSelector options={[
+                      {label: "Published on", field:"download.date", order: "desc", defaultOption: true},
+                      {label: "Build date", field:"build.date", order: "desc"}
+                    ]}/>
+                  </ActionBarRow>
+
+                  <ActionBarRow>
+                    <SelectedFilters/>
+                    <ResetFilters/>
+                  </ActionBarRow>
+
+                  <MenuFilter field="source.product"
+                              title="Product"
+                              id="products"
+                              listComponent={Tabs}
+                              translations={{"All":"All products"}}/>
```

Comment:
Nit: missing space after column in object definition. Also, shouldn't we use [prettier](https://github.com/prettier/prettier) to format code and avoid wasting time with CS bikeshedding?